packages/bonsai/bonsai.v0.15.1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "Jane Street developers"
+authors: ["Jane Street Group, LLC"]
+homepage: "https://github.com/janestreet/bonsai"
+bug-reports: "https://github.com/janestreet/bonsai/issues"
+dev-repo: "git+https://github.com/janestreet/bonsai.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/bonsai/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"                    {>= "4.08.0"}
+  "async"                    {>= "v0.15" & < "v0.16"}
+  "async_extra"              {>= "v0.15" & < "v0.16"}
+  "async_js"                 {>= "v0.15" & < "v0.16"}
+  "async_kernel"             {>= "v0.15" & < "v0.16"}
+  "async_rpc_kernel"         {>= "v0.15" & < "v0.16"}
+  "async_rpc_websocket"      {>= "v0.15" & < "v0.16"}
+  "core"                     {>= "v0.15" & < "v0.16"}
+  "core_bench"               {>= "v0.15" & < "v0.16"}
+  "core_kernel"              {>= "v0.15" & < "v0.16"}
+  "core_unix"                {>= "v0.15" & < "v0.16"}
+  "expect_test_helpers_core" {>= "v0.15" & < "v0.16"}
+  "fuzzy_match"              {>= "v0.15" & < "v0.16"}
+  "incr_dom"                 {>= "v0.15" & < "v0.16"}
+  "incr_map"                 {>= "v0.15" & < "v0.16"}
+  "patdiff"                  {>= "v0.15" & < "v0.16"}
+  "ppx_css"                  {>= "v0.15" & < "v0.16"}
+  "ppx_jane"                 {>= "v0.15" & < "v0.16"}
+  "ppx_pattern_bind"         {>= "v0.15" & < "v0.16"}
+  "ppx_typed_fields"         {>= "v0.15" & < "v0.16"}
+  "profunctor"               {>= "v0.15" & < "v0.16"}
+  "textutils"                {>= "v0.15" & < "v0.16"}
+  "virtual_dom"              {>= "v0.15" & < "v0.16"}
+  "cohttp-async"             {>= "2.5.6" & < "3.0.0" | >= "5.0.0"}
+  "dune"                     {>= "2.0.0"}
+  "gen_js_api"               {>= "1.0.8"}
+  "js_of_ocaml-ppx"          {>= "3.9.0"}
```

Comment:
I'm not sure why my changes in the upstream opam file weren't taken here. Where are you taken those opam files from? Any chance you could just use `opam publish` ?
```suggestion
  "js_of_ocaml"              {>= "4.0.0"}
  "js_of_ocaml-ppx"          {>= "4.0.0"}
```

Reply:
I've been doing it manually. (I don't think I *can* use opam publish.)